galette/templates/default/elements/list.html.twig

```diff
@@ -134,7 +134,7 @@
         $(function(){
     {% if batch is defined and batch_actions is defined and batch_actions|length > 0 %}
             var _init_page = function(res){
-        {% if login.isAdmin() or login.isStaff() %}
+        {% if login.isAdmin() or login.isStaff() or login.isGroupManager() %}
```

Comment:
Well, this seems to be enough to fix batch actions for group managers in core and existing plugins (https://github.com/galette/plugin-events/pull/30 is required for plugin-events).

Please, have a deeper look to check the possible deleted parts of code you were talking about previously, and change the draft status of this PR at your will.

I checked as admin, staff, group manager and member, and ACLs and rights settings appear to be respected in each case everywhere batch actions are available (members, contributions, transactions, cars models, cars, bookings).

Let me know if I missed anything :blush:

Reply:
I think https://github.com/galette/galette/pull/306/commits/e028062a43dd45ee8cff314284717a4d4ec1e893 is enough; if you're OK with that, we can merge this one :)

Reply:
I'm OK with that :+1: :rocket: